source/features/open-all-notifications.tsx

```diff
@@ -67,26 +67,31 @@ function addOpenReposButton(): void {
 	}
 }
 
-function addOpenAllButton(className: string, text: string): void {
-	// Selector works on:
-	// https://github.com/notifications (Grouped by date)
-	// https://github.com/notifications (Grouped by repo)
-	// https://github.com/notifications?query=reason%3Acomment (which is an unsaved filter)
-	select('.js-check-all-container .js-bulk-action-toasts ~ div .Box-header')!.append(
-		<button className={'btn btn-sm d-none ' + className} type="button">
-			<LinkExternalIcon className="mr-1"/>{text}
-		</button>,
-	);
-}
+// Selector works on:
+// https://github.com/notifications (Grouped by date)
+// https://github.com/notifications (Grouped by repo)
+// https://github.com/notifications?query=reason%3Acomment (which is an unsaved filter)
+const notificationHeaderSelector = '.js-check-all-container .js-bulk-action-toasts ~ div .Box-header';
+
+const openUnreadButtonClass = 'rgh-open-notifications-button';
```

Comment:
This className is also used in another JSX above; that one  should use this variable too.